test/version-tolerance/initial-tests/rlcClient.spec.ts

```diff
@@ -0,0 +1,47 @@
+import RLCClient, { LLCClientRestClient } from "../generated/llc-initial/src";
+import { assert } from "chai";
+
+const phase = "initial";
+
+describe(`RLC Version Tolerance ${phase} phase`, async () => {
```

Comment:
will the inital-tests/rlcClient.spec.ts and updated-test/rlcClient.specs.ts be mostly the same for non breaking change scenarios ?

Reply:
Yes they are the same, the only thing that changes is the import. This way if there is a breaking change the updated test will fail